backend/variables/builtin/readFile.js

```diff
@@ -12,19 +12,36 @@ const model = {
     definition: {
         handle: "readFile",
         usage: "readFile[\"path/to/file.txt\"]",
-        description: "Read contents of a text file. Optionally include 'true' as a second argument to select a random line.",
+        description: "Read contents of a text file. Optionally include 'true' as a second argument to select a random line, or a number to read a specific line.",
         possibleDataOutput: [OutputDataType.TEXT]
     },
     evaluator: (_, filePath, randomLine) => {
 
         if (filePath === null || !filePath.endsWith(".txt")) return "[File Path Error]";
         try {
             let contents = fs.readFileSync(filePath, "utf8");
-            if (randomLine) {
+            let filteredLines = [];
+
+            if (randomLine === "true" || !isNaN(randomLine)) {
```

Comment:
If the user just inputs `true` for the second variable, it will show up as a boolean, not a string, because the variable system is suppah smart (thanks Reject!), so we need to account for that.

```js
let shouldReadRandomLine = randomLine === true || randomLine === "true";
if (shouldReadRandomLine || !isNaN(randomLine)) {
```